dist/pom.xml

```diff
@@ -17,332 +17,1519 @@
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <launchers>
+                                <launcher>
+                                    <id>update_rapids_config</id>
+                                    <mainClass>com.nvidia.spark.rapids.RapidsConf</mainClass>
+                                    <args>
+                                        <arg>${project.basedir}/../docs/configs.md</arg>
+                                    </args>
+                                </launcher>
+                            </launchers>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>update_supported</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <launchers>
+                                <launcher>
+                                    <id>update_rapids_support</id>
+                                    <mainClass>com.nvidia.spark.rapids.SupportedOpsDocs</mainClass>
+                                    <args>
+                                        <arg>${project.basedir}/../docs/supported_ops.md</arg>
+                                    </args>
+                                </launcher>
+                            </launchers>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>update_supported_tools</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <launchers>
+                                <launcher>
+                                    <id>update_rapids_support_tools</id>
+                                    <mainClass>com.nvidia.spark.rapids.SupportedOpsForTools</mainClass>
+                                    <args>
+                                        <arg>${project.basedir}/../tools/src/main/resources/supportedDataSource.csv
+                                        </arg>
+                                    </args>
+                                </launcher>
+                            </launchers>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>com.nvidia.spark.rapids.SparkShimServiceProvider.*</exclude>
```

Comment:
the shade plugin outputs  `dependency-reduced-pom.xml` under the pom's basedir by default since we were not using `dependencyReducedPomLocation` config. As a consequence many developers who have used the old build in their local repo prior to this PR will have a stale file under dist that is not cleaned up by maven

I suggest to add it to the exclude list in this PR to minimize reported build failures.

In a follow up we can add dependencyReducedPomLocation to the  shade plugin configs